### PR TITLE
R0ad qb jobs

### PR DESCRIPTION
**Pull request by r0adra93**
Incremental update
Boss Menu Works
Multi Job Menu getting there
Vehicle Menu Broken
Clothing Menu Broken
Armory Broken

### CODE REVIEW

**Review by Irishstevie — APPROVED** [review]
Yea this is good to go as its still a WIP By Road alone

**Review by Irishstevie — APPROVED** [review]
LGTL

File: jobs/bus.lua [hunk 1]
```diff
@@ -1,17 +1,58 @@
-local QBCore = exports['qb-core']:GetCoreObject()
 Config.Jobs = Config.Jobs or {}
 Config.Jobs.bus = {
     ["label"] = "Bus Driver",
+    ["webHooks"] = {
+        ["bus"] = "https://discord.com/api/webhooks/1068882053430517823/1_GeFJUZPbdI15MPoX0Iq8PgUE8BINl3OO74vV3J6c2tJXUJsHMTeDrVu1mbY_dZWD3P"
```
Comment (zwkf) [inline jobs/bus.lua:5]: Don't include secrets in your code

File: jobs/garbage.lua [hunk 2]
```diff
@@ -1,17 +1,58 @@
-local QBCore = exports['qb-core']:GetCoreObject()
 Config.Jobs = Config.Jobs or {}
 Config.Jobs.garbage = {
     ["label"] = "Sanitation Engineer",
+    ["webHooks"] = {
+        ["garbage"] = "https://discord.com/api/webhooks/1068882053430517823/1_GeFJUZPbdI15MPoX0Iq8PgUE8BINl3OO74vV3J6c2tJXUJsHMTeDrVu1mbY_dZWD3P"
```
Comment (zwkf) [inline jobs/garbage.lua:5]: Remove secret

File: jobs/lawyer.lua [hunk 3]
```diff
@@ -1,34 +1,75 @@
 --Place into into qb-legal keep seperate config from judge--
-local QBCore = exports['qb-core']:GetCoreObject()
 Config.Jobs = Config.Jobs or {}
 Config.Jobs.lawyer = {
     ["label"] = "Law Firm",
+    ["webHooks"] = {
+        ["lawyer"] = "https://discord.com/api/webhooks/1068882053430517823/1_GeFJUZPbdI15MPoX0Iq8PgUE8BINl3OO74vV3J6c2tJXUJsHMTeDrVu1mbY_dZWD3P"
```
Comment (zwkf) [inline jobs/lawyer.lua:6]: Remove secret

File: jobs/lawyer.lua [hunk 4]
```diff
@@ -1,34 +1,75 @@
...
+    },
     ["defaultDuty"] = true,
     ["offDutyPay"] = false,
     ["inCityHall"] = {
         ["listInCityHall"] = true, -- true he job is sent to city hall | false the job is not in city hall
         ["isManaged"] = true -- true the job is sent to the boss of the job | false the job is automatically assigned
     },
+    ["menu"] = {
+        ["icons"] = {
```
Comment (zwkf) [inline jobs/lawyer.lua:15]: Do we need to keep repeating these icons?

File: jobs/mechanic.lua [hunk 5]
```diff
@@ -1,33 +1,74 @@
-local QBCore = exports['qb-core']:GetCoreObject()
 Config.Jobs = Config.Jobs or {}
 Config.Jobs.mechanic = {
     ["label"] = "mechanic",
+    ["webHooks"] = {
+        ["mechanic"] = "https://discord.com/api/webhooks/1068882053430517823/1_GeFJUZPbdI15MPoX0Iq8PgUE8BINl3OO74vV3J6c2tJXUJsHMTeDrVu1mbY_dZWD3P"
```
Comment (zwkf) [inline jobs/mechanic.lua:5]: As above

File: jobs/police.lua [hunk 6]
```diff
@@ -1,7 +1,9 @@
-local QBCore = exports['qb-core']:GetCoreObject()
 Config.Jobs = Config.Jobs or {}
 Config.Jobs.police = { -- name the job
     ["label"] = "Law Enforcement", -- label that display when typing in /job
+    ["webHooks"] = {
+        ["police"] = "https://discord.com/api/webhooks/1068882053430517823/1_GeFJUZPbdI15MPoX0Iq8PgUE8BINl3OO74vV3J6c2tJXUJsHMTeDrVu1mbY_dZWD3P"
```
Comment (zwkf) [inline jobs/police.lua:5]: As above

Comment (r0adra93) [reply jobs/police.lua]: No secret is a dead link

**Pull request closed without merging**